docs/license.rst

```diff
@@ -0,0 +1,24 @@
+License
+=======
+
+The MIT License (MIT)
```

Comment:
Is there some way to include [this license file]( https://github.com/zarr-developers/zarr-python/blob/main/LICENSE ) without copying it?

Reply:
Ah sorry, let me clarify. Is there a way with Sphinx to tell it to load the content from `LICENSE` as opposed to creating a second copy of it here? IOW would it be possible to drop this `rst` file somehow?

Reply:
No worries. Can see how that might be unclear.

Maybe this [SO question is helpful]( https://stackoverflow.com/q/2921724 )? We could also add a symlink if being outside `docs` is an issue

Reply:
from https://www.sphinx-doc.org/en/master/usage/markdown.html
I think `myst-parser`has to be installed as well

Reply:
Good morning @jakirkham. I wasn't able to completely remove the license.rst file but i could link it to the original LICENSE file without copying the content.
I referenced this issue https://github.com/sphinx-doc/sphinx/issues/7000

Reply:
Actually this is exactly what I was looking for. Thank you! 😄